tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html

```diff
@@ -547,8 +576,10 @@
           if (selection) {
             brushedExtent[colIndex] =
                 calculateBrushedExtent(colIndex, selection);
+            _this._brushSelections[colIndex] = selection;
```

Comment:
Can you help me understand what selection is? Comment in L573 says it is an extent but what does it mean to call, `d3.select(axis).call(selection)`, the selection in L608?